types/vimeo__vimeo/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "private": true,
+    "name": "@types/vimeo__vimeo",
+    "version": "3.0.9999",
+    "projects": [
+        "https://github.com/vimeo/vimeo.js"
+    ],
+    "devDependencies": {
+        "@types/vimeo__vimeo": "workspace:."
+    },
+    "owners": [
+        {
+            "name": "Adam Thompson-Sharpe",
+            "githubUsername": "MysteryBlokHed"
+        }
+    ]
```

Comment:
Like I mentioned in the PR body, I can add the original owner from the vimeo package here.
cc. @mattleff